website/content/en/applications.adoc

```diff
@@ -3,41 +3,57 @@ title: "About Applications for FreeBSD"
 sidenav: about
 ---
 
-= About Applications for FreeBSD
+= Applications for FreeBSD
 
-== Experience the possibilities with FreeBSD
+FreeBSD goes beyond what's commonly expected of a UNIX(R) workstation.
 
-FreeBSD can handle nearly any task you would expect of a UNIX(R) workstation, as well as many you might not expect:
+== Thousands of applications
 
-== FreeBSD is a true open system with full source code.
+There are more than thirty thousand link:../ports/[ports] and related packages.
 
-There is no doubt that so-called open systems are _the_ requirement for today's computing applications. But no commercial vendor-supplied solution is more open than one which includes full source code to the entire operating system, including the kernel and all of the system daemons, programs, and utilities. You can modify any part of FreeBSD to suit your personal, organizational, or corporate needs.
+Non-ported software will often compile and run with ease.
+FreeBSD has https://freebsdfoundation.org/freebsd/timeline/[UNIX and BSD heritage].
 
-With its generous link:../copyright/freebsd-license/[licensing policy], you can use FreeBSD as the basis for any number of free _or commercial_ applications.
+== Use cases
 
-== FreeBSD runs thousands of applications.
+*Internet services*.
+FreeBSD is famed for performant, reliable networking for Internet service providers.
+Ported software includes https://nginx.org/[nginx] and https://ws.apache.org/[Apache Web Services](TM).
+FTP requirements can be fulfilled with ports of http://proftpd.org/[ProFTPD] and https://security.appspot.com/vsftpd.html[vsftpd].
 
-Because FreeBSD is based on 4.4BSD, an industry-standard version of UNIX, it is easy to compile and run programs. FreeBSD also includes an extensive packages collection and link:../ports/[ports collection] that bring precompiled and easy-to-build software right to your desktop or enterprise server. There is also a growing number of link:../commercial/software/[commercial applications] written for FreeBSD.
+*Networking*.
+Packet filtering, routing, DNS, firewalling, email hosting, file and print services.
 
-Here are some examples of the environments in which FreeBSD is used:
+*Desktop*.
+Ported desktop environments include https://xfce.org/[Xfce], https://lxde.org/[LXDE], https://kde.org/plasma-desktop/[KDE Plasma] and https://www.gnome.org[GNOME].
+There's a metaport for https://x.org/[X.Org](TM).
+Supported libraries include industry-standard https://www.opengl.org/[OpenGL](R) and https://motif.ics.com/[Motif].
```

Comment:
I would certainly mention wx, gtk (3 and 4) and Qt (5 and 6) here, since those are used a bit more often now than Motif.

Reply:
[`a0eaee8` (#103)](https://github.com/freebsd/freebsd-doc/pull/103/commits/a0eaee83a1f2756b6cf094d585813c31462d2f22)

Reply:
A bit :-)